core/cog-view.c

```diff
@@ -111,12 +118,26 @@ cog_view_class_init(CogViewClass *klass)
         g_param_spec_boolean("use-key-bindings", NULL, NULL, TRUE,
                              G_PARAM_READWRITE | G_PARAM_CONSTRUCT | G_PARAM_EXPLICIT_NOTIFY | G_PARAM_STATIC_STRINGS);
 
+    /**
+     * CogView:viewport: (default-value null)
+     *
+     * Viewport where the CogView belongs.
+     *
+     * Since: 0.20
+     */
+    s_properties[PROP_VIEWPORT] =
+        g_param_spec_object("viewport", NULL, NULL, G_TYPE_OBJECT, G_PARAM_READABLE | G_PARAM_CONSTRUCT);
```

Comment:
Let's also have the `G_PARAM_STATIC_NAME` flag here, so GLib won't copy the `"viewport"` string:

```suggestion
        g_param_spec_object("viewport", NULL, NULL, G_TYPE_OBJECT, G_PARAM_READABLE | G_PARAM_CONSTRUCT | G_PARAM_STATIC_NAME);
```